config/prod.exs

```diff
@@ -10,7 +10,9 @@ use Mix.Config
 # which you should run after static files are built and
 # before starting your production server.
 config :smart_home_auth, SmartHomeAuthWeb.Endpoint,
-  url: [host: "example.com", port: 80],
+  http: [port: {:system, "PORT"}],
+  url: [scheme: "https", host: "dwylauth.herokuapp.com", port: 443],
```

Comment:
Will this cause issues deploying elsewhere/ within a home as a `hub`?

Reply:
good catch. It won't cause issues, but it's worth updating. (_I've updated it_) ✅